internal/services/synapse/synapse_managed_private_endpoint_resource.go

```diff
@@ -147,11 +162,11 @@ func resourceSynapseManagedPrivateEndpointRead(d *pluginsdk.ResourceData, meta i
 			d.SetId("")
 			return nil
 		}
-		return fmt.Errorf("retrieving Synapse Managed Private Endpoint (Workspace %q / Resource Group %q): %+v", id.WorkspaceName, id.ResourceGroup, err)
+		return fmt.Errorf("retrieving %s: %+v", id, err)
```

Comment:
this is a parsed ID, so it'll be returned as a pointer - as such this needs to be:

```suggestion
		return fmt.Errorf("retrieving %s: %+v", *id, err)
```